docs/meshcloud.payment-methods.md

```diff
@@ -0,0 +1,94 @@
+---
+id: meshstack.payment-methods
+title: Payment Methods
+---
+
+## Introduction
+
+Just like you pay for a new t-shirt online via your credit card, enterprises also have to be responsible for the costs that they make in cloud platforms. Payment methods are used for allocating cloud costs, all the way back to the responsible department of your organization, for a clear and transparent accounting process. As the cloud can be mighty with its enormous offering of practically unlimited services, it is important that all meshProjects and meshCustomers are correctly charged and that this information makes its way back to the required systems. Payment methods therefore are the link between the cloud usage and the resulting cost in the context of a meshProject.
+
+Payment methods are created **per** meshCustomer and come in two variants:
+
+1) A cost center payment method (indicated via `COST_CENTER`), meaning the costs made by the meshTenants in your meshProjects will be fully charged to the cost center of your organization.
+
+2) A budget payment method (indicated via `COST_LIMITATION`), meaning costs made by the meshTenants in your meshProjects will be charged based on the monetary budget amount that is set on this payment method.
+
+![Payment Methods Lifecycle](assets/payment_methods/payment_method_overall_lifecycle.png)
+
+## The Payment Method Lifecycle: Creation
+
+The first step in reaping the benefits of payment methods is creating them. A few important things must be considered about payment methods before creating them
+
+1. A payment method is **always** scoped to a single meshCustomer. This means it is not possible to create a global payment method that can be used for all meshCustomers. It is also not possible to re-use a single payment method across multiple meshCustomers.
+2. A payment method's identifier is globally unique. This means you cannot create a payment method with the same identifier for multiple meshCustomers. Make sure to use an identifier that will not collide in other meshCustomers.
+
+### Creating a payment method via the meshPanel
+
+One way of creating payment methods is via the meshPanel. To do so, make sure that you have an account with the 'Partner Admin' role. Navigate to the Admin area and follow these steps:
+
+1. Click on 'Customers' on the left.
+2. Find a customer you want to create a payment method for and click on the 'Customer Payment Methods' icon on the right.
+
+    ![List Payment Methods](assets/payment_methods/customer_list_payment_methods.png)
+
+3. Click on 'Create Payment Method' at the top right.
+4. Enter a name and identifier for the new payment method.
+5. Choose a type for the payment method. Read more here on what type to choose and what the individual differences are.
+6. (This is optional) Set a maximum amount of EUR on the payment method to indicate the remaining budget for this year.
```

Comment:
d: do we only support EUR as currency?

Reply:
good point as we also support $ this is an inconsistency in the product. Good catch

Reply:
So this is a product gap right? I won't have to change the docs

Reply:
I will add it to the cost management initiative